### PR TITLE
convert libva related packages to use meson

### PR DESCRIPTION
**Pull request by lrusak**
self explanatory.

libva-utils loses a couple utilities but nothing we care about.

### CODE REVIEW

**Comment by lrusak** [comment]
@MilhouseVH updated

File: packages/multimedia/intel-vaapi-driver/package.mk [hunk 1]
```diff
@@ -11,15 +11,17 @@ PKG_SITE="https://01.org/linuxmedia"
...
 PKG_LONGDESC="intel-vaapi-driver: VA-API user mode driver for Intel GEN Graphics family"
-PKG_TOOLCHAIN="autotools"
+PKG_TOOLCHAIN="meson"
 
 if [ "$DISPLAYSERVER" = "x11" ]; then
-  DISPLAYSERVER_LIBVA="--enable-x11 --disable-wayland"
+  DISPLAYSERVER_LIBVA="-Dwith_x11=yes -Dwith_wayland=no"
 elif [ "$DISPLAYSERVER" = "weston" ]; then
-  DISPLAYSERVER_LIBVA="--disable-x11 --enable-wayland"
+  DISPLAYSERVER_LIBVA="-Dwith_x11=yes -Dwith_wayland=yes"
```
Comment (Kwiboo) [inline packages/multimedia/intel-vaapi-driver/package.mk]: `-Dwith_x11=yes` looks wrong for weston (untested)

```suggestion
  DISPLAYSERVER_LIBVA="-Dwith_x11=no -Dwith_wayland=yes"
```

Comment (lrusak) [reply packages/multimedia/intel-vaapi-driver/package.mk]: Thanks!

**Comment by lrusak** [comment]
Updated

**Pull request merged**